### PR TITLE
Standardise versioning numbers

### PR DESCRIPTION
**Pull request by acm19**
Documents standard for version numbers and how the should be updated.

Resolves: https://github.com/acm19/aws-request-signing-apache-interceptor/issues/49

### Pull Request Checklist:

- [x] I have added a contribution line at the top of [CHANGELOG.md](CHANGELOG.md).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -24,3 +24,11 @@ make release
 Once a release is prepared 2 Pull Request will be generated. They have to be merged in order. First the one for the release and then the one for the next development version (it should be a draft PR). The automation will take care of the rest.
```
Comment (dblock) [inline RELEASING.md:24]: Lowercase pull request

File: RELEASING.md [hunk 2]
```diff
@@ -24,3 +24,11 @@ make release
...
 
 > **_NOTE:_** Please review the changes in case there are errors if the PR were modified after the automation ran.
+
+#### Versioning standard
```
Comment (dblock) [inline RELEASING.md:28]: For consistency capitalize Standard or just leave it as Versioning

File: RELEASING.md [hunk 3]
```diff
@@ -24,3 +24,11 @@ make release
...
+
+Releases should follow the following convention for versioning numbers. It's responsibility of the committer to update the version number within the same pull request where the change was introduced when the changes affect `minor` or `major` as follows:
```
Comment (dblock) [inline RELEASING.md:30]: I think you should at least point this to https://semver.org/ and optionally copy-paste their version. So I would start with "This project follows [semver](https://semver.org/).".

File: RELEASING.md (hunk 1 above)
Comment (acm19) [reply RELEASING.md]: fixed

File: RELEASING.md (hunk 2 above)
Comment (acm19) [reply RELEASING.md]: Removed.

File: RELEASING.md [hunk 4]
```diff
@@ -17,10 +17,23 @@ Go the project root directory and run
 make release
 ```
 
-> **_NOTE:_** If you are not a maintainer, you might want to set the `REMOTE` env variable to your preferred remote name. The `create_pull_requests` target might fail and you will need to create the PRs manually from the branches that were pushed.
+> **_NOTE:_** If you are not a maintainer, you might want to set the `REMOTE` env variable to your preferred remote name. The `create_pull_requests` target might fail, then you will need to create the PRs manually from the branches that were pushed.
 
 ### For Maintainers
 
-Once a release is prepared 2 Pull Request will be generated. They have to be merged in order. First the one for the release and then the one for the next development version (it should be a draft PR). The automation will take care of the rest.
+Once a release is prepared 2 pull request will be generated. They have to be merged in order. First the one for the release and then the one for the next development version (it should be a draft PR). The automation will take care of the rest.
```
Comment (dblock) [inline RELEASING.md:24]: Typo: requests.

```
Once a release is prepared, 2 pull requests will be generated. They have to be merged in order. First, the one for the release, and then the one for the next development iteration (it should be a draft PR). The automation will take care of the rest.
```

File: RELEASING.md [hunk 5]
```diff
@@ -17,10 +17,23 @@ Go the project root directory and run
...
 
 > **_NOTE:_** Please review the changes in case there are errors if the PR were modified after the automation ran.
+
+#### Versioning
+
+This project follows (semver)[https://semver.org/] versioning standards.
```
Comment (dblock) [inline RELEASING.md:30]: "versioning standards" is redundant :) Semver = versioning standard.

**Review by dblock — APPROVED** [review]
I'm nitpicking. Feel free to merge at will.

**Comment by acm19** [comment]
> I'm nitpicking. Feel free to merge at will.

I like to pay attention to details, I'm not in a hurry to merge :)

File: RELEASING.md (hunk 4 above)
Comment (acm19) [reply RELEASING.md]: Fixed.

File: RELEASING.md (hunk 5 above)
Comment (acm19) [reply RELEASING.md]: I've made it explicit now.

**Review by dblock — APPROVED** [review]

**Pull request merged**